app/views/users/index.html.erb

```diff
@@ -1,164 +1,101 @@
-      quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.</p>
-    <p>Fusce bibendum in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin
-      cursus, nulla quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.Fusce bibendum
-      in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin cursus, nulla
-      quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.</p>
-    <p>Fusce bibendum in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin
-      cursus, nulla quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.Fusce bibendum
-      in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin cursus, nulla
-      quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.</p>
-    <p>Fusce bibendum in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin
-      cursus, nulla quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.Fusce bibendum
-      in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin cursus, nulla
-      quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.</p>
-    <p>Fusce bibendum in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin
-      cursus, nulla quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.Fusce bibendum
-      in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin cursus, nulla
-      quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.</p>
-
-  </div>
-</section>
-
-
-<footer>
-  <p> Copyright &copy; the best team </p>
-</footer>
-<!-- New -->
-<!-- My driver -->
-<!-- Shared with me --> <!-- Sorted by date-->
-<!-- Recent --> <!-- File which were opened recently -->
-<!-- Starred -->
-<!-- _________ -->
-<!-- Storage --> <!-- Not clickable, showing just how much storage is left-->
-
-<!-- My Drive with the options of creating folders etc. -->
-<!-- List view/ Grid view-->
-<!-- Details about the recent activity or details about the current drive or the current folder-->
-<!-- _________________ -->
-<!-- Folders -->
-<!-- Files -->
-<!-- Organized -->
-
 
+<body>
+	<div class="container">
+		<header>
+			<div class="toolbar" role="banner">
+			  <img width="40" alt="Logo" src="images/logo.png" class="header-image">
+  				<span>Universal Storage Tool</span>
+    				<div class="spacer"></div>
+				<div class="topnav">
+					<label>
+						<input type="text" placeholder="Search..">
+					</label>
+				</div>
+    			<img src="https://picsum.photos/50" alt="account info" class="header-image">
```

Comment:
If you want to display the real avatar of the account use something like  
``` 
<%= image_tag current_user.avatar_url, class: 'placeholder', style: 'placeholder' %>
```
You can use just the image_tag till the first coma, but you might want to use styling to make it round like you did so that's why I left these so you won't have trouble with chaining the syntax. ;*